classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize
```

Comment:
- [OPTIONAL] The `@label instance variable, should be initialized in the constructor` since it can take different values, something like so: `def initialize(label)`, please fix that. Checkout this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/class_and_object_for_irb.md#constructor-and-instance-vars) for more clarifications.